exporter/opentelemetry-exporter-zipkin-json/src/opentelemetry/exporter/zipkin/json/__init__.py

```diff
@@ -83,6 +85,7 @@
 from opentelemetry.exporter.zipkin.node_endpoint import IpInput, NodeEndpoint
 from opentelemetry.sdk.environment_variables import (
     OTEL_EXPORTER_ZIPKIN_ENDPOINT,
+    OTEL_EXPORTER_ZIPKIN_TIMEOUT,
```

Comment:
Should this env var be proposed to the specs?

Reply:
PR in spec repo https://github.com/open-telemetry/opentelemetry-specification/pull/1636. It has already two approvals and I don't see any objection coming.

Reply:
Nice looks like it merged!